Documentation/install/upgrade.rst

```diff
@@ -320,6 +320,50 @@ IMPORTANT: Changes required before upgrading to 1.8.0
   .. note:: The ENI IPAM mode automatically derives the native routing CIDR so
             no action is required.
 
+Upgrading from >=1.7.0 to 1.8.y
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Since Cilium 1.5, the conntrack table size parameter
+   ``bpf-ct-global-tcp-max`` was set to the default value ``1000000`` to retain
+   backwards compatibility with previous versions. In Cilium 1.8 the default
+   value is set to 512K by default in order to reduce the agent memory
+   consumption. This means at maximum 512K concurrent TCP connections across all
+   endpoints are supported by default. To prevent from breaking established
+   connections, ``bpf-ct-global-tcp-max`` must be manually set to
+   ``1000000`` (or any other sufficiently large value) before upgrading.
+
+   To check whether any action is required the following command can be used to
+   check the currently configured maximum number of entries:
+
+   .. code:: bash
+
+      sudo bpf-map info /sys/fs/bpf/tc/globals/cilium_ct4_global | grep "Max entries:"
```

Comment:
> FWIW if we wanted to avoid `bpf-map` we could just check the headers we use to compile the datapath:
> 
> ```
> # grep -R CT_MAP_SIZE_TCP /var/run/cilium/state/templates/
> /var/run/cilium/state/templates/942ab6c4186d8b5f243c0f6fa59341122f671be1/lxc_config.h:#define CT_MAP_SIZE_TCP 524288
> ```

Good point, using that command instead.

Reply:
> I'm not sure how to tell the equivalent for the NAT table size since I didn't have an instance with BPF nodeport lying around to test. Probably `SNAT_MAPPING_IPV4_SIZE`?

Yes, looks like it's `SNAT_MAPPING_IPV4_SIZE` and/or `SNAT_MAPPING_IPV6_SIZE` depending on IPv4/IPv6 being enabled/disabled. Not checked on an actual instance though, but by checking the sources.

Reply:
Will add the corresponding `grep` command as well.